ocs_ci/deployment/deployment.py

```diff
@@ -652,22 +652,74 @@ def deploy_ocs_via_operator(self, image=None):
         logger.info("Creating namespace and operator group.")
         run_cmd(f"oc create -f {constants.OLM_YAML}")
 
-        # create multus network
+        # Create Multus Networks
```

Comment:
are we doing the same also for nodes added post deployment?

Reply:
I haven't added anything regarding nodes added post-deployment. I suspect we would want to enable PROMISC on the network interface for the newly added node. Are we running add_node tests in vsphere environments?

Reply:
Yes, add node tests are done on vSphere (I believe both UPI and IPI).
@clacroix12 @OdedViner should we handle the adjustments needed for Multus in add node functionality with this PR or a follow up one?

Reply:
@ebenahar missed this update earlier but yes I think any changes necessary for test cases to work with multus deployments should be done in isolated PRs.